.swiftlint.yml

```diff
@@ -2,7 +2,6 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - AEPAudience/Tests
```

Comment:
SwiftLint should run on the Tests as well, so this line can be removed and the Xcode project setup to run SwiftLint the the build phase of the tests. We can do this in another PR if desired as the lint changes may touch a lot of files.